packages/api-derive/src/balances/all.ts

```diff
@@ -32,8 +33,8 @@ function calcBalances ([accountId = EMPTY_ACCOUNT, bestNumber = ZERO, [freeBalan
 
   // offset = balance locked at genesis, perBlock is the unlock amount
   const { offset, perBlock } = vesting.unwrapOr(new VestingSchedule());
-  const vestedNow = perBlock.mul(bestNumber);
-  const vestedBalance = vestedNow.gt(offset)
+  const vestedNow: BN = perBlock.mul(bestNumber);
```

Comment:
Can be for another PR, but I'm wondering why in api-derive are we putting BNs everywhere? surely BlockNumber and Balance are more semantically correct

Reply:
Good point! But I'd also leave that for another PR since it means touching all files & tests